packages/abi-gen/README.md

```diff
@@ -102,3 +102,23 @@ yarn clean
 ```bash
 yarn lint
 ```
+### CLI tests
+
+The files in `test-cli/` are used to test the CLI output against a set of dummy contracts. 
+
+Compile dummy contracts and generate wrappers:
```

Comment:
Can we scrub out any new mentions of "dummy"?  Right now it's true that all we have is "a dummy contract," but we're planning on expanding the test cases in there.  If we'd like to have a word/phrase that describes the contract(s) as "not real," might I suggest "test fixture" or simply "fixture."

Going further, and adding on to my top-level comment (trying to choose a folder hierarchy that can be applied to any package providing a command-line utility), I suggest we keep all the contracts and artifacts (and maybe even tsconfig.json?) under a folder called `test-cli/fixtures`.

Reply:
tsconfig.json can only reference files in subdirectories so it has to stay in the highest level directory of all files it touches. 

But I agree that we can check in the artifacts and move them to a fixtures folder.